docs/wiki/services/list-of-services.md

```diff
@@ -10,22 +10,24 @@
 ### Active trading but no withdraw/deposit
 
 - [BitForex](https://www.bitforex.com/)
-- [HitBTC](https://hitbtc.com/)
+- [HitBtc](https://hitbtc.com/)
 - [Hotbit](https://www.hotbit.io/)
 - [KuCoin](https://www.kucoin.com/)
-- [Kaiserex](https://www.kaiserex.com/)
-- [BigONE](https://big.one/en)
+- [BKEX](https://www.bkex.com/trade/GRIN_USDT)
+- [ZB](https://www.zb.com/en/kline/grin_usdt)
 
 ## Block Explorers
 
 - [Grinscan](https://grinscan.net/)
 - [Grinmint](https://grinmint.com/explorer/)
 - [GrinExplorer](https://grinexplorer.net/)
+- [ScanGrinMoney](https://scan.grin.money/)
 
 ## Mining Pools
 
 - [GrinMint](https://www.grinmint.com)
-- [WoolyPooly](https://woolypooly.com)
```

Comment:
It seems WoolyPooly has something about Grin here https://woolypooly.com/en/coin/grin . Does this no longer work?

Apart from the minor question, this seems good :+1:

Reply:
Grin mining is offline more than 1 month period, mwc C31 focused now. But can stay, in case they make a return. Thanks